toonz/sources/toonzlib/txsheet.cpp

```diff
@@ -1395,8 +1395,8 @@ void searchAudioColumn(TXsheet *xsh, std::vector<TXshSoundColumn *> &sounds,
     TXshColumn *column = xsh->getColumn(i);
     if (column) {
       TXshSoundColumn *soundCol = column->getSoundColumn();
-      if (soundCol && ((isPreview && soundCol->isPreviewVisible()) ||
-                       (!isPreview && soundCol->isCamstandVisible()))) {
+      if (soundCol && ((isPreview && soundCol->isCamstandVisible()) ||
+                       (!isPreview && soundCol->isPreviewVisible()))) {
```

Comment:
NOTE: This comment is not a modification request.
I confirmed your solution works fine. But realized that the `SoundProperties::m_isPreview` works opposite as I expected - when this value is true, which means the sound column is on scrubbing or on playing with preview button is OFF... Hmm, it's confusing.

Reply:
Yeah. I thought the same thing. Decided not to change it because other places had their own "isPreview" and felt it better to maintain consistency. Would have been better that the Preview toggle was called Render Toggle instead. ☺